test/molecule-role/molecule/default/prepare.yml

```diff
@@ -213,6 +265,27 @@
         CODE_NAME: "{{ agent_current_branch }}"
         SKIP_SSL_VALIDATION: "true"
 
+- name: Prepare Connection Namespaces VM - Agent (post)
+  hosts: agent_linux_connection_namespace_vm
```

Comment:
this can go away, because the group if the new vm is `agent_linux_vm` the agent will be installed on it tru the `Prepare Linux Agents (post)` play